### DISCUSSION TITLE
Open a new window?

### DISCUSSION THREAD

**Discussion by hesyifei** [gdh1995/vimium-c#617] [Q&A] [answered]
We have `createTab` to open a new tab. Is there a command for opening a new window? Thanks!

**Comment by gdh1995** [comment, marked as answer]
`map xxx openUrl reuse="new-window" url="......"`

**Reply by hesyifei** [reply]
Thanks! What if I want to open a new window with focus on the address bar (like Ctrl+N)?

**Reply by gdh1995** [reply]
`map xxx createTab reuse="new-window"`

**Reply by gdh1995** [reply]
In fact, `createTab` works almost the same as `openUrl`.

Removing `url="..."` will make Chrome decide what to focus, and then it will choose the address bar
